src/vma/proto/route_entry.h

```diff
@@ -39,6 +39,8 @@
 
 #include "vma/proto/route_rule_table_key.h"
 #include "vma/infra/cache_subject_observer.h"
+#include "vma/dev/net_device_val.h"
+#include "vma/dev/net_device_table_mgr.h"
```

Comment:
g_p_net_device_table_mgr from net_device_table_mgr.h is used in side both route_entry.cpp and route_table_mgr.cpp so I moved it to those file and replace both net_device_val.h and net_device_table_mgr.h with net_device_entry.h.